articles/quickstart/webapp/aspnet-core/v2/03-user-profile.md

```diff
@@ -56,11 +56,11 @@ public IActionResult Profile()
 }
 ```
 
-The `User.Identity.Name` property will look for a claim of type `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name` on the user object. Auth0 passes the name of the user in the `name` claim of the `id_token`, but this does not get automatically matched the the `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name` claim type, and therefore `User.Identity.Name` will return null.
+The `User.Identity.Name` property looks for a claim of a type `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name` on the user object. Auth0 passes the name of the user in the `name` claim of the ID token. The user's name is not matched to this claim type, which means that `User.Identity.Name` will return null.
```

Comment:
No. You are completely changing the meaning of this paragraph. Please change back to the way it was. Or if you need to change it, make sure it keeps its meaning. What you have written here does not make sense!